build-scripts/ubuntu-1604/postinst_node

```diff
@@ -120,6 +120,49 @@ TimeoutSec=300
 WantedBy=multi-user.target
 EOF
 
+# add supervisord script
+cat <<EOF > /etc/supervisor/indy-node.conf
+[supervisord]
+nodaemon=true
+logfile=/var/log/indy/supervisord.log
+logfile_maxbytes=10MB
+loglevel=critical
+
+[supervisorctl]
+serverurl=http://127.0.0.1:9001
+
+[inet_http_server]
+port = 127.0.0.1:9001
+
+[rpcinterface:supervisor]
+supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
+
+[program:indy-node]
+command=/usr/bin/env python3 -O /usr/local/bin/start_indy_node %(ENV_NODE_NAME)s %(ENV_NODE_PORT)s %(ENV_NODE_CLIENT_PORT)s
```

Comment:
Could you please clarify how NODE_NAME, NODE_PORT and NODE_CLIENT will be available in env? Have I missed something in supervisor's logic?

Reply:
Fixed